### DISCUSSION TITLE
Should we add flight information to the form?

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#14] [Q&A] [answered]
Right now only the date of arrival is on the form.

I feel by adding the arrival time that we could use that to better staff during busier times and predict when more/less personnel would be required.

**Comment by judyn808** [comment]
This sounds like a Phase 2+ idea. AQS doesn't sound like they're ready to optimize their employee scheduling process yet.

**Reply by JJ-Foust** [reply]
Sorry some of these were added before we got any feedback from AQS.

**Comment by JJ-Foust** [comment, marked as answer]
They are not wanting to change their process at this time.
